src/coreclr/vm/ClrEtwAll.man

```diff
@@ -1775,6 +1775,30 @@
                         </UserData>
                     </template>
 
+                    <template tid="WaitStart_V1">
```

Comment:
I've named the event just Wait. MonitorWait doesn't really fit because the name Monitor is not used where the event is emitted. ContentionWait could be confused with Contention.

Reply:
I'd suggest we call it MonitorWaitStart. 'Wait' alone feels overly broad. I wouldn't worry about SyncBlock not using the term 'Monitor' in the source. SyncBlock is the name of the internal implementation detail and Monitor is the abstraction that .NET devs see. I assume the goal here is to give an event whenever Monitor.Wait() occurs, regardless of how the runtime might change the implementation internally so naming it after the abstraction seems accurate to me.

I'd also suggest dropping the V1. The convention is starting at version 0 with no suffix. Higher numbered versions are used when creating future revisions to the event.